torchmultimodal/utils/common.py

```diff
@@ -216,3 +217,7 @@ def custom_forward(*inputs: Any) -> Callable:
             return fn(cls, *inputs, **kwargs)
 
     return inner
+
+
+def get_clones(module: nn.Module, n: int) -> nn.Module:
+    return nn.ModuleList([deepcopy(module) for i in range(n)])
```

Comment:
Does PyTorch officially state that `deepcopy`-ing `nn.Module` keep working for future?

Reply:
Thanks for the comment. I haven't noticed this before. Would you mind sharing some details?
For context, `deepcopy` is used by PT core's own implementation to stack up transformer layers [here](https://pytorch.org/docs/stable/_modules/torch/nn/modules/transformer.html#TransformerDecoder).

Reply:
I cannot find the `__deepcopy__` protocol implemented on [torch.nn.Module](https://github.com/pytorch/pytorch/blob/master/torch/nn/modules/module.py). The multiple Q&A just says it works. So I was wondering if it's tested and ensured in PyTorch core.